include/libopencm3/stm32/f4/pwr.h

```diff
@@ -72,14 +73,13 @@ LGPL License Terms @ref lgpl_license
 
 /* --- Function prototypes ------------------------------------------------- */
 
-enum pwr_vos_scale {
-	PWR_SCALE1,
-	PWR_SCALE2,
-};
+#define	PWR_SCALE1	0x3
+#define PWR_SCALE2	0x2
+#define PWR_SCALE3	0x1
```

Comment:
hrm? what's worng with the enum?  we use an enum on all the other platforms with multi bits, eg f7, l1, l0, (stopped looking)